src/models/EventHostMapper.php

```diff
@@ -56,6 +56,18 @@ public function addHostToEvent($event_id, $host_id)
         return $this->_db->lastInsertId();
     }
 
+    public function removeHostfromEvent($host_id, $event_id)
+    {
+        $sql = 'DELETE FROM user_admin WHERE uid = :user_id AND rid = :event_id AND rtype = :type';
+        $stmt = $this->_db->prepare($sql);
+
+        return $stmt->execute([
+            ':host_id'  => $host_id,
```

Comment:
They query contains :user_id, but the params has :host_id - so this throws an error :-)